src/deployment/windows.md

```diff
@@ -0,0 +1,136 @@
+
+Joining the [Microsoft Partner Network][microsoftpartner] is also required.
+
+## Set up your application in the Partner Center
+
+Manage an application's life cycle in the [Microsoft Partner Center][microsoftpartner].
+
+First off, it is necessary to reserve the application name and ensure that the 
+required rights to the name exist. Once the name is reserved, the application 
+will be provisioned for services such as push notifications, and it is possible
+to start adding add-ons.
+
+Options such as pricing, availability, age ratings, and category have to be 
+configured together with the first submission and are automatically retained 
+for the subsequent submissions.
+
+## Packaging and deployment
+
+In order to publish an application to Microsoft Store,
+it first has to be packaged. The valid formats are **.msix**, **.msixbundle**,
+**.msixupload**, **.appx**, **.appxbundle**, **.appxupload**, and **.xap**.
+
+### Manual packaging and deployment
+
+Check out [MSIX packaging][msix packaging] to learn about packaging 
+Flutter Windows Desktop applications.
+
+Note that each product has a unique identity, which the Store assigns.
+
+If the package is being built manually, it is necessary to include its 
+identity details manually during the packaging. The essential information
+can be retrieved from the Partner Center:
+
+1. Navigate to the application in the Partner Center.
+2. Select **Product management**.
+3. Retrieve the package identity name, publisher, 
+and publisher display name by clicking on **Product identity**.
+
+After manually packaging the application, it will also have to be 
+manually submitted to the [Microsoft Partner Center][microsoftpartner].
+This can be done by creating a new submission, navigating to **Packages**, 
+and uploading the created application package.
+
+### Continuous deployment
+
+In addition to manually creating and deploying the package, 
+it is possible to automate the build, package, versioning, 
+and deployment process using CI/CD tooling after having submitted
+the application to the Microsoft Store for the first time.
+
+#### Codemagic CI/CD
+
+[Codemagic CI/CD][codemagic] uses the [`msix` pub package][msix package] to package 
+Flutter Windows Desktop applications. 
+
+For Flutter applications, either the [Codemagic Workflow Editor][cmworkfloweditor] or [codemagic.yaml][cmyaml] 
+can be used to package the application and deploy to Microsoft Partner Center.
+Additional options (such as the list of capabilites and language resources 
+contained in the package) can be configured using the aforementioned package.
+
+For publishing, Codemagic uses the [Partner Center submission API][partnercenterapi]; thus,
+Codemagic requires [associating the Azure Active Directory and Partner Center accounts][azureadassociation].
+
+## Updating the app's version number
+
+With Flutter Windows Desktop, the version number must be set during the 
+packaging process and can not be set via the `pubspec.yaml` 
```

Comment:
We should probably document how to set this here.

Context: https://github.com/flutter/website/issues/6466